### DISCUSSION TITLE
I don't understand

### DISCUSSION THREAD

**Discussion by Pierre-Thibault** [RaphGL/Tuckr#45] [Q&A] [answered]
Can you explain what this program does and how to use it because I understand nothing about what you said in the doc?

For example, if I do:

`tuckr add .zshrc`

I have this error:

`.zshrc doesn't exist.`

But the file is there in my home directory.

**Comment by RaphGL** [comment, marked as answer]
`tuckr add` adds a group (symlinks it to home), it doesn't work on a file by file basis. 

A group is just a directory in `.config/dotfiles/Configs` or the equivalent in your operating system with dotfiles in it. 
For example for zsh you could have a directory with the path `.config/dotfiles/Configs/zsh`, which is then referred as the `zsh` group on tuckr.

The file grouping is there so that you can put all dotfiles for a single program together and then have the program validate that all dotfiles necessary for a program are correctly deployed. 

Instead of doing the setup you can just use `tuckr push <group> <file...>` to add all related files and directories to that group. Then can do `tuckr status` to see if they've been symlinked or are in conflict. 
To symlink them you just do `tuckr add <group>`.

If you want an example of tuckr being used, check my own dotfiles: https://github.com/raphgl/dotfiles

---
I'm not sure if you read only the readme or not, but there's a wiki I wrote attempting to explain how it works: https://github.com/RaphGL/Tuckr/wiki

**Comment by Pierre-Thibault** [comment]
Thank you RaphGL for this explanation. I use stow now.

**Reply by RaphGL** [reply]
Alrighty, no problem :)
